server/tornado_app.py

```diff
@@ -0,0 +1,88 @@
+import functools
+
+from tornado import websocket, web, ioloop
+from tornado.gen import coroutine
+
+import rethinkdb as r
+
+from init_db import get_bigchain
+
+clients = []
+bigchain = get_bigchain()
+
+# from http://blog.hiphipjorge.com/django-and-realtime-using-django-with-tornado-and-rethinkdb/
+r.set_loop_type("tornado")
+
+
+@coroutine
+def print_changes(db_table):
+    conn = yield bigchain.conn
+    feed = yield r.table(db_table).changes().run(conn)
+    while (yield feed.fetch_next()):
+        change = yield feed.next()
+        block = get_block_from_change(change, db_table)
+        for client in clients:
+            for tx in block:
+                if tx_contains_vk(tx['transaction'], client.username):
```

Comment:
OK, then we should probably put a TODO note here, since in essence these examples should tell users how to use bigchaindb.